sklearn/linear_model/passive_aggressive.py

```diff
@@ -110,6 +121,16 @@ def partial_fit(self, X, y, classes=None):
         -------
         self : returns an instance of self.
         """
+        if self.class_weight == 'balanced':
+            raise ValueError("class_weight 'balanced' is not supported for "
+                             "partial_fit. In order to use 'balanced' "
+                             "weights, use "
+                             "compute_class_weight('balanced', classes, y). "
```

Comment:
Should this error message indicate that `compute_class_weight` can be found in `sklearn.utils`?